liquibase-core/src/main/java/liquibase/util/FileUtil.java

```diff
@@ -67,4 +69,16 @@ public static void write(String contents, File file) throws IOException {
             StreamUtil.copy(new ByteArrayInputStream(contents.getBytes(LiquibaseConfiguration.getInstance().getConfiguration(GlobalConfiguration.class).getOutputEncoding())), output);
         }
     }
+
+    public static String getFileNotFoundMessage(String physicalChangeLogLocation) {
```

Comment:
Ticky-tacky comment:  there is an extra space in the message string between the word 'by' and the word 'absolute'

Specifying files by  absolute path
                            ^^

Reply:
Thanks for catching that. Fixed it up